tests/rdf_format_converter.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
Is this script just required to convert ttl test data to all other formats?

Reply:
yep, `ttl` is our default format, and this script is just used to generate test sample files.

Reply:
Maybe you can add a one line comment explaining what the script does (and that it is for internal use mainly).

Reply:
Thanks, updated!